keep/api/core/db.py

```diff
@@ -38,6 +38,7 @@
 from keep.api.models.db.rule import *  # pylint: disable=unused-wildcard-import
 from keep.api.models.db.tenant import *  # pylint: disable=unused-wildcard-import
 from keep.api.models.db.workflow import *  # pylint: disable=unused-wildcard-import
+from keep.api.models.provider import ProviderAlertsCountDTO
```

Comment:
better not import dtos in here, return the .count() number and construct the response in the route itself